docs/modules/ROOT/pages/api/multitoken.adoc

```diff
@@ -0,0 +1,257 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:multiToken-guide: xref:multiToken.adoc[MultiToken guide]
+:erc1155-metadata: xref:https://eips.ethereum.org/EIPS/eip-1155#metadata[ERC1155Metadata]
+
+= MultiToken
+
+This module provides the full MultiToken module API.
+
+TIP: For an overview of the module, read the {multiToken-guide}.
+
+== Core
+
+[.hljs-theme-dark]
+```ts
+import "./node_modules/@openzeppelin-compact/multi-token/MultiToken" prefix MultiToken_;
+```
+
+[.contract]
+[[MultiToken]]
+=== `++MultiToken++` link:https://github.com/OpenZeppelin/compact-contracts/blob/main/contracts/multiToken/src/MultiToken.compact[{github-icon},role=heading-link]
+
+[.contract-index]
+.Circuits
+--
+
+[.sub-index#MultiTokenModule]
+* xref:#MultiTokenModule-initialize[`++initialize(uri_)++`]
+* xref:#MultiTokenModule-uri[`++uri(id)++`]
+* xref:#MultiTokenModule-balanceOf[`++balanceOf(account, id)++`]
+* xref:#MultiTokenModule-setApprovalForAll[`++setApprovalForAll(operator, approved)++`]
+* xref:#MultiTokenModule-isApprovedForAll[`++isApprovedForAll(account, operator)++`]
+* xref:#MultiTokenModule-transferFrom[`++transferFrom(from, to, id, value)++`]
+* xref:#MultiTokenModule-_transfer[`++_transfer(from, to, id, value)++`]
+* xref:#MultiTokenModule-_update[`++_update(from, to, id, value)++`]
+* xref:#MultiTokenModule-_unsafeTransferFrom[`++_unsafeTransferFrom(from, to, id, value)++`]
+* xref:#MultiTokenModule-_unsafeTransfer[`++_unsafeTransfer(from, to, id, value)++`]
+* xref:#MultiTokenModule-_setURI[`++_setURI(newURI)++`]
+* xref:#MultiTokenModule-_mint[`++_mint(to, id, value)++`]
+* xref:#MultiTokenModule-_unsafeMint[`++_unsafeMint(to, id, value)++`]
+* xref:#MultiTokenModule-_burn[`++_burn(from, id, value)++`]
+* xref:#MultiTokenModule-_setApprovalForAll[`++_setApprovalForAll(owner, operator, approved)++`]
+--
+
+[.contract-item]
+[[MultiTokenModule-initialize]]
+==== `[.contract-item-name]#++initialize++#++(uri_: Opaque<"string">) → []++` [.item-kind]#circuit#
+
+Initializes the contract by setting the base URI for all tokens.
+
+This MUST be called in the implementing contract's constructor.
+Failure to do so can lead to an irreparable contract.
+
+Requirements:
+
+- Contract is not initialized.
+
+[.contract-item]
+[[MultiTokenModule-uri]]
+==== `[.contract-item-name]#++uri++#++(id: Uint<128>) → Opaque<"string">++` [.item-kind]#circuit#
+
+This implementation returns the same URI for *all* token types.
+It relies on the token type ID substitution mechanism defined in the EIP: {erc1155-metadata}.
```

Comment:
Add on to above change
```suggestion
It relies on the token type ID substitution mechanism defined in the EIP: {erc1155-metadata}[ERC1155-Metadata].
```

Reply:
I moved `[ERC1155-Metadata]` back to the attribute declaration at the top